### PR TITLE
[SKIP-644] Remove unsafe inputs

### PR DESCRIPTION
**Pull request by anderssonw**
Inputs and GitHub contexts may be used to inject malicious code in workflows. This PR removes any direct usage of an input in bash scripts and other contexts, for consistency.

I have not sanitised run-trivy completely as it is removed in #14.

### CODE REVIEW

**Review by Flexman94 — APPROVED** [review]

**Comment by esphen** [comment]
Think you may have missed some. Have a look at `get-attestation` for example

**Comment by anderssonw** [comment]
> Think you may have missed some. Have a look at `get-attestation` for example

Aye, fixed :)

**Comment by anderssonw** [comment]
vet ikke om 6f284e6 var nødvendig men kanskje likesågreit å være konsekvent?

File: .github/workflows/post-build-attest.yml [hunk 1]
```diff
@@ -33,23 +35,23 @@ jobs:
...
             digest=$(docker manifest inspect $image_tag_url -v | jq -r .Descriptor.digest)
             base_image_url=${image_tag_url%:*}
             image_digest_url="${base_image_url}@${digest}"
           fi
           echo "image_url=$image_digest_url" >> $GITHUB_OUTPUT
-      
+
       - name: Check if already attested
         id: get-attestation
         run: |
```
Comment (odasko) [inline .github/workflows/post-build-attest.yml:57]: Tror denne trenger env PROJECT_ID og IMAGE_ID

File: .github/workflows/post-build-attest.yml [hunk 2]
```diff
@@ -33,23 +35,23 @@ jobs:
         with:
           workload_identity_provider: ${{ inputs.workload_identity_provider }}
```
Comment (odasko) [inline .github/workflows/post-build-attest.yml:36]: env var i 'Authenticate with Google Cloud' stegene også? 😆

**Pull request merged**